ckan/views/group.py

```diff
@@ -1109,6 +1121,8 @@ def post(self, group_type, is_organization, id=None):
 
         try:
             group_dict = _action(u'group_member_create')(context, data_dict)
+            h.flash_success(_(u'Assigned %s as %s.') % (
+                data_dict['username'], data_dict['role']))
```

Comment:
role can't be translated this way, use different `_(strings)` based on the role instead?

Reply:
Using `authz.trans_role` now. does that work??

Reply:
Wow, I hate the way trans_role is written, and I bet it's going to result in incorrect capitalization, but I guess it works

Reply:
Hahaha, just making these changes to the upstream PR too. Looks like `trans_role` is written differently there: https://github.com/ckan/ckan/blob/master/ckan/authz.py#L269

Reply:
Yeah looks like Sergey cleaned it up. But still, `trans_role` returns the wrong capitalization (and maybe other problems like masculine/feminine) to insert into a sentence

Reply:
Yeah, would be nice to have a full humanization implementation for roles

Reply:
Could use a dict lookup here to solve the issue. i.e
`{'member': _(u'Assigned %s as a member.'), ...}`

Reply:
Yeah, reckon that is the best thing to do for now. I have done that here and upstream then